pkg/version/version.go

```diff
@@ -1,4 +1,12 @@
 package version
 
+import (
+	"fmt"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+)
+
 // Version number of the CLI
 var Version = "master"
```

Comment:
It might be worthwhile to note here that this value may be overwritten by Goreleaser, and that's how the final version value is generated (IMO, the fact that Go's compiler let's you do this is not obvious at all). Here's the comment I had in stripe-mock for inspiration:

```
// This is set to the actual version by GoReleaser (using `-ldflags "-X ..."`)
// as it's run. Versions built from source will always show master.
```

Reply:
Good call, that could definitely be confusing otherwise. Updating!